images/go-ipfs/tests/docker-test.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+CONTAINER_NAME="ipfs-host-$(uuidgen)"
+CONTAINER_PORT=${FREE_PORT}
+REQUEST_RETRIES=1
+RETRY_DELAY=15
+ipfs_staging=$(mktemp -d)
```

Comment:
I would get rid of this - it is just going to make getting at the test file from in the container more annoying.